setup/ansible/roles/common_ubuntu/tasks/sys_deps.yml

```diff
@@ -15,45 +14,13 @@
   with_items:
     - git
     - build-essential
-    - python
-    - python-setuptools
-    - python-pip
-    - python-virtualenv
     - cmake
-    - swig
     - wget
-    - cmake
```

Comment:
cmake is still required for building libswiftnav - is that handled by something in the libswiftnav build dependencies in the provisioning?